homeassistant/components/reolink/host.py

```diff
@@ -68,6 +68,13 @@ async def async_init(self) -> bool:
         if self._api.mac_address is None:
             return False
 
+        if not self._api.is_admin:
+            _LOGGER.warning(
```

Comment:
Maybe we should create a repair issue instead with a flow that reloads the integration after the user has changed to an admin user and we've checked that that is working. The users that want to continue with a non admin user can ignore the repair issue. That we way we don't log a warning every time the integration is reloaded.

This assumes we want to allow users to keep using non admin user. If not, we should start a re-auth flow.

Reply:
I do not think we schould allow users to use non-admin user, it will create problems wich will only generate a lot of user issues.
I really do not want to deal with all those issues.

I already had people telling me multiple times that the ONVIF port was enabled, guess what: the ONVIF port was not enabled....
You cannot see if it is enabled or enable it through the web interface, mobile application or windows client, only option is through API (with admin privilages) or using a mouse and screen connected to the NVR directly.

Besides changing anything like turning on the spotlight, controlling PTZ, turn on/off notifications etc all require admin.